### PR TITLE
updated maintainer information

### PR DESCRIPTION
**Pull request by jameslamb**
Updating `DESCRIPTION` to use my personal email.

### CODE REVIEW

**Review by austin3dickey — APPROVED** [review]

File: r-pkg/DESCRIPTION [hunk 1]
```diff
@@ -3,7 +3,7 @@ Type: Package
 Title: Get Data Frame Representations of 'Elasticsearch' Results
 Version: 0.3.1
 Authors@R: c(
-    person("James", "Lamb", email = "james.lamb@uptake.com", role = c("aut", "cre")),
+    person("James", "Lamb", email = "jaylamb20@gmail.com", role = c("aut", "cre")),
```
Comment (jameslamb) [inline r-pkg/DESCRIPTION]: missed the maintainer field!

**Comment by jameslamb** [comment]
Not the first time I've seen this "codecov is commenting on a closed-for-many-years PR" problem.

@austin3dickey I think we should just rip out codecov. This repo is so lightly maintained, I don't think we're getting much from it, and one less integration is one less source of noise and security issues.

I'll put up a PR this week.

**Comment by austin3dickey** [comment]
Yeah, in 2026 each of those links smells like a social engineering scam 😅

**Comment by jameslamb** [comment]
Strong agree, I thought the same thing.

**Pull request merged**